src/main/kotlin/no/nav/familie/ef/sak/oppgave/OppgaveService.kt

```diff
@@ -438,4 +424,7 @@ class OppgaveService(
             Oppgavetype.VurderHenvendelse -> false
             else -> error("Håndterer ikke behandlesAvApplikasjon for $oppgavetype")
         }
+
+    private fun lagOpprettOppgavebeskrivelse(beskrivelse: String?) =
+        "--- ${dagensDatoMedTidNorskFormat()} opprettet --- \n${beskrivelse.orEmpty()}"
```

Comment:
Kan vi fjerne opprettet i "overskriften"?
Bør vi å ha med systemnavn her? Sjelden saksbehandlere oppretter oppgaver direkte i vår løsning, er jo som regel en konsekvens av at noe annet har skjedd. 

Som diskutert i slack - hvis beskrivelse er tom - Burde det bare stå "Oppgave opprettet"?

Reply:
Da går jeg for løsning som legger seg opp mot gosys: 

--- dd.MM.yyyy hh:mm familie-ef-sak ---
[HVA] (med default oppgave opprettet hvis beskrivelse er null) 

Legger ikke på "saksbehandler id" i metadata, men setter familie-ef-sak. Man kunne kanskje diskutert om oppgave skulle vært opprettet av innlogget saksbehanldler dersom behandling er opprettet manuelt, men jeg tror ikke det er nødvendig.